src/clusterfuzz/_internal/protos/uworker_msg.proto

```diff
@@ -118,22 +108,28 @@ message AnalyzeTaskOutput {
   optional string platform_id = 17;
 }
 
+message CrashInfo{
+  optional bool is_new = 1;
+  optional int64 count = 2;
+  optional string crash_type = 3;
+  optional string crash_state = 4;
+  optional bool security_flag = 5;
+}
+
 message FuzzTaskOutput {
-  optional string fuzzer_name = 1;
+  optional string fully_qualified_fuzzer_name = 1;
   optional string crash_revision = 2;
   optional float job_run_timestamp = 3;
   optional int64 new_crash_count = 4;
   optional int64 known_crash_count = 5;
   optional int64 testcases_executed = 6;
-  optional Json job_run_crashes = 7;
-  optional string fully_qualified_fuzzer_name = 8;
+  repeated CrashInfo job_run_crashes = 7;
   optional int32 new_targets_count = 9;
 }
 
 message MinimizeTaskOutput {
-  optional Json last_crash_result_dict = 1;
+  map<string, string> last_crash_result_dict = 1;
```

Comment:
address is a field here that seems to be an int based on some of your other files, so not sure we can say this is always a string.
Maybe we should leave a json field and have helper methods in uworker_io (name them copy_dict_from_json_field and save_dict_to_json_field to make clear that updating the dict wont work without calling again).

Reply:
this field is only populated with the return of `_extract_crash_result`
crash_address is a string as defined in CrashInfo class here:
https://github.com/google/clusterfuzz/blob/6877dd76c50cc5b25807f91d8534a31f125197e1/src/clusterfuzz/stacktraces/__init__.py#L34